tests/integration/conftest.py

```diff
@@ -139,9 +141,15 @@ def __call__(self) -> ContextManager[ProtostarFixture]:
 
 @pytest.fixture(name="create_protostar_project", scope="module")
 def create_protostar_project_fixture(
+    request: FixtureRequest,
     session_mocker: MockerFixture,
     tmp_path_factory: TempPathFactory,
 ) -> CreateProtostarProjectFixture:
+    cairo_version = (
+        CairoVersion.cairo0 if not hasattr(request, "param") else request.param
+    )
+    assert isinstance(cairo_version, CairoVersion)
```

Comment:
If request doesn't have an attribute "param" we default to CairoVersion.cairo0 - this way we don't break any existing tests.